src/pages/boundary/downloads/index.tsx

```diff
@@ -1,26 +1,9 @@
-import { ReactElement } from 'react'
-import { GetStaticProps } from 'next'
 import boundaryData from 'data/boundary.json'
-import installData from 'data/boundary-install.json'
 import { ProductData } from 'types/products'
-import { generateStaticProps, GeneratedProps } from 'lib/fetch-release-data'
 import ProductDownloadsView from 'views/product-downloads-view'
+import { generateGetStaticProps } from 'views/product-downloads-view/server'
 
-const BoundaryDownloadsPage = (props: GeneratedProps): ReactElement => {
-	const { latestVersion, releases } = props
-	return (
-		<ProductDownloadsView
-			latestVersion={latestVersion}
-			pageContent={installData}
-			releases={releases}
-		/>
-	)
-}
+const getStaticProps = generateGetStaticProps(boundaryData as ProductData)
 
-export const getStaticProps: GetStaticProps = async () => {
-	const product = boundaryData as ProductData
-
-	return generateStaticProps(product)
-}
-
-export default BoundaryDownloadsPage
+export { getStaticProps }
```

Comment:
⛏️ nit: this could be:

```ts
export const getStaticProps = generateGetStaticProps(boundaryData as ProductData)
```

Reply:
Thanks! I prefer having all exports in one place at the bottom of a file, so that's the context on why I've taken this approach.